### PR TITLE
Update all the bindings

### PR DESCRIPTION
**Pull request by Beyley**
# Summary of the PR
t���wJD�����              �!�~�-<[Y�
zIDYuL\F�O��޾��c��榦

# Related issues, Discord discussions, or proposals

# Further Comments
�'8TL+▒uI7�6��z5��▒rP2��c0�wK :|�eO��I▒�$N���!����A[��c�+"���E��]6�C�m��?▒�~��$Z��6Y�$I��q���n�h�����u���B�x�����u�jc�
����Fx ▒��[?��4b�{�~���▒L���"�0▒

### CODE REVIEW

**Comment by Perksey** [comment]
Just checking in on the status here? Maybe we can get a checklist going?

**Comment by Perksey** [comment]
The progress on native builds is awesome btw, just SwiftShader now which isn't the most urgent thing in the world so we can leave out for this update (Vulkan is Vulkan, doesn't really spoil...)

File: src/SPIRV/Silk.NET.SPIRV/Enums/SourceLanguage.gen.cs [hunk 1]
```diff
@@ -36,6 +36,8 @@ public enum SourceLanguage : int
         Wgsl = 0xA,
         [NativeName("Name", "SpvSourceLanguageSlang")]
         Slang = 0xB,
+        [NativeName("Name", "SpvSourceLanguageZig")]
+        Zig = 0xC,
```
Comment (Perksey) [inline src/SPIRV/Silk.NET.SPIRV/Enums/SourceLanguage.gen.cs:40]: haha nice

File: src/WebGPU/Extensions/Silk.NET.WebGPU.Extensions.Dawn/Structs/PfnRequestAdapterCallback.gen.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 #pragma warning disable 1591
 
-namespace Silk.NET.WebGPU
+namespace Silk.NET.WebGPU.Extensions.Dawn
```
[the remark this answers is not in the document]
Comment (Beyley) [reply src/WebGPU/Extensions/Silk.NET.WebGPU.Extensions.Dawn/Structs/PfnRequestAdapterCallback.gen.cs:17]: i have no idea, its probably a SilkTouch regression *somehow*?

**Comment by alexrp** [comment]
> just SwiftShader now which isn't the most urgent thing in the world

I've already fixed SwiftShader in my branch FWIW. I don't think it needs to block this PR.

**Comment by Perksey** [comment]
Awesome, thanks.

File: src/WebGPU/Extensions/Silk.NET.WebGPU.Extensions.Dawn/Structs/PfnRequestAdapterCallback.gen.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Perksey) [reply src/WebGPU/Extensions/Silk.NET.WebGPU.Extensions.Dawn/Structs/PfnRequestAdapterCallback.gen.cs]: This has been fixed. WebGPU-Native and Dawn have diverged and Dawn is still using the old names. Added some missing renaming logic for pfn types to allow us to make them share the same name and therefore not be added to the extension project.

**Comment by Perksey** [comment]
@Beyley Is this ready for review if this build passes? (I've already reviewed most of the files tbh!)

**Review by Perksey — APPROVED** [review]

**Pull request merged**